friends/src/component/Person.js

```diff
@@ -0,0 +1,35 @@
+import React from "react"
+
+
+const Person = (props) =>{
```

Comment:
I really like that you went so far as to add the functionality that you could click on a single friend and see just them based on their id.